### PR TITLE
docs: misc as weblate made me look this up (smoe:docs_misc)

### PR DESCRIPTION
**Pull request by smoe**
Reviewed a series of already translated items that weblate had issues with.

### CODE REVIEW

File: docs/src/gcode/tool-compensation.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ tool table automatically.
 
 Typical steps for updating the tool table:
 
-* After homing load a tool with 'Tn M6' where 'n' is the tool number.
+* After homing load a tool with 'T__n__ M6' where _n_ is the tool number.
```
Comment (silopolis) [inline docs/src/gcode/tool-compensation.adoc:21]: Why all those double underscores ?

**Comment by silopolis** [comment]
Le mar. 6 sept. 2022 à 15:16, Steffen Möller ***@***.***> a
écrit :

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In docs/src/gcode/tool-compensation.adoc
> <https://github.com/LinuxCNC/linuxcnc/pull/1992#discussion_r963696066>:
>
> > @@ -18,7 +18,7 @@ tool table automatically.
>
>  Typical steps for updating the tool table:
>
> -* After homing load a tool with 'Tn M6' where 'n' is the tool number.
> +* After homing load a tool with 'T__n__ M6' where _n_ is the tool number.
>
> You need them to get a fraction of a word in italic. The single letter 'T'
> is fixed, *n* the variable.
>

Nice trick! Thanks 👍🙏

**Comment by hansu** [comment]
> Nice trick! Thanks 👍🙏 #1530 style guide

This is a general asciidoc thingy, so nothing linuxcnc specific, and therefore nothing for the style IMHO.

**Comment by smoe** [comment]
Rebased after conflict with master.

**Comment by smoe** [comment]
From my understanding, it is only the upload of the generated PDFs that is failing for buster and bookworm.

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> From my understanding, it is only the upload of the generated PDFs
> that is failing for buster and bookworm.

I suspect the issue is that we are uploading too much after the CI
builds started building on several Debian releases.  Perhaps drop the
deb and pdf upload until we can figure out what is going on?

-- 
Happy hacking
Petter Reinholdtsen

**Comment by smoe** [comment]
> [Steffen Möller]
> From my understanding, it is only the upload of the generated PDFs that is failing for buster and bookworm.
> I suspect the issue is that we are uploading too much after the CI builds started building on several Debian releases. Perhaps drop the deb and pdf upload until we can figure out what is going on?

Or have the PDF only for the latest release?

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> Or have the PDF only for the latest release?

Yes, reducing the amount is probably a good idea.  What do you mean by
'latest release', sid?

-- 
Happy hacking
Petter Reinholdtsen

**Comment by smoe** [comment]
Yes, sid or bookworm.

**Comment by smoe** [comment]
Rebase, squash and reupload to trigger new build.

**Comment by smoe** [comment]
@petterreinholdtsen , would you have insights on why the CI builds fail? I don't understand what is happening.

File: docs/src/index.tmpl [hunk 2]
```diff
@@ -262,13 +262,13 @@ function setup_page(){
...
 		<li><a href="gcode/tool-compensation.html">Tool Compensation</a></li>
 		<li><a href="gui/tooledit.html">Tool Table Editor</a></li>
-		<li><a href="gcode/overview.html">G-Code Overview</a></li>
-		<li><a href="gcode/g-code.html">G-Codes</a></li>
-		<li><a href="gcode/m-code.html">M-Codes</a></li>
-		<li><a href="gcode/o-code.html">O-Codes</a></li>
+		<li><a href="gcode/overview.html">G-code Overview</a></li>
+		<li><a href="gcode/g-code.html">G-codes</a></li>
+		<li><a href="gcode/m-code.html">M-codes</a></li>
+		<li><a href="gcode/o-code.html">O-codes</a></li>
```
Comment (hansu) [inline docs/src/index.tmpl:268]: Not sure if this should be uppercase in a heading?

**Comment by silopolis** [comment]
Le lun. 3 oct. 2022 à 19:06, Hans Unzner ***@***.***> a
écrit :

> ***@***.**** commented on this pull request.
>
> Not sure if this should be uppercase in a heading?
>

Looks good to me

**Comment by hansu** [comment]
Thanks for changing the titles according to my wish :slightly_smiling_face:

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> @petterreinholdtsen , would you have insights on why the CI builds
> fail? I don't understand what is happening.

Only that it seem apt repository related, and Sebastian was looking into
it.  No idea what is actually wrong, nor how to fix it.

-- 
Happy hacking
Petter Reinholdtsen

**Pull request merged**